pom.xml

```diff
@@ -60,7 +65,13 @@
         <dependency>
             <groupId>com.github.onsdigital</groupId>
             <artifactId>dp-jwt-verifier-java</artifactId>
-            <version>0.1.2</version>
+            <version>0.5.0</version>
+        </dependency>
+
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-core</artifactId>
+            <version>2.15.0-rc2</version>
```

Comment:
This dependency comes from dp-logging which is under our control. We should update dp-logging instead of overriding it here. However, this is still a release candidate so maybe worth waiting for a proper release? 
I think we should temporarily ignore this vulnerability and add this repo to this existing ticket: https://trello.com/c/esmoaPFm